package Protos;

import "base.proto";

message STDSynapseC {
    required double psp_decay = 1;
    required double amp = 2;
    required double gamma = 3;
    required double tau_d = 4;
}

message STDSynapseState {
    required double res = 1;
}

